crates/runtime/src/secrets.rs

```diff
@@ -304,3 +312,44 @@ pub fn get_secret_or_param(
 
     None
 }
+
+#[allow(unused_imports)]
```

Comment:
```suggestion
```

Reply:
I added this because Clippy seems to be incorrectly flagging this
```
   --> crates/runtime/src/secrets.rs:317:9
    |
317 |     use std::collections::HashMap;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `-D unused-imports` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(unused_imports)]`
error: unused imports: `Secret`, `get_secret_or_param`
   --> crates/runtime/src/secrets.rs:319:26
    |
319 |     use crate::secrets::{get_secret_or_param, Secret};
    |                          ^^^^^^^^^^^^^^^^^^^  ^^^^^^
```